### ISSUE TITLE
Match-case causes syntax errors

### ISSUE THREAD

**Issue by nefrob** [charliermarsh/ruff#2377]
Ex.

```python
x = 1
match x:
    case 1:
        pass

E999 SyntaxError: invalid syntax. Got unexpected token 'x'
```

when running `ruff --fix` on `v0.0.238` and with `target-version = "py311"` in `pyproject.toml`.

**Comment by messense** [comment]
Duplicate of https://github.com/charliermarsh/ruff/issues/282

**Comment by charliermarsh** [comment]
Just closing as duplicate.

**Comment by charliermarsh** [comment]
Fixed as of v0.0.250.
